src/js/containers/router/RouterRoutes.jsx

```diff
@@ -317,7 +330,7 @@ const routeDefinitions = {
         },
         {
             path: 'detachedHelp',
-            onEnter: checkHelpUserPermissions,
+            onEnter: checkUserPermissions,
```

Comment:
It seemed unnecessary to let users access the FABS help page if they don't have access to the FABS data in general, so it bumps them to the default Help. I wanted to keep the functionality as similar as possible to what it was before FABS was added to the Broker.